src/main/java/net/imglib2/algorithm/util/Grids.java

```diff
@@ -213,6 +213,7 @@ public static void forEachOffset(
 	 * Get all blocks of size {@code blockSize} contained within an interval
 	 * specified by {@code dimensions} and their positions within a cell grid.
 	 *
+	 * @implNote op name='grid.partition', type=Function
```

Comment:
Should these be Ops? To which namespace should they belong?

Reply:
Let's hold off on the grid functions for now.